app/views/travel_advice/country.html.erb

```diff
@@ -30,7 +30,11 @@
 
 </main>
 
-<div id="related-items"></div>
+<% if @navigation_helpers %>
```

Comment:
The country page is only used from `private-frontend`, a "preview" version of this app running in the backend, only visible to editors. I think the easiest thing would be to not show the sidebar here.